tests/python_package_test/test_plotting.py

```diff
@@ -14,164 +13,188 @@
-        self.assertIsInstance(ax2, matplotlib.axes.Axes)
-        self.assertEqual(ax2.get_title(), '')
-        self.assertEqual(ax2.get_xlabel(), '')
-        self.assertEqual(ax2.get_ylabel(), '')
+            "num_leaves": 3}
+
+
+@pytest.mark.skipif(not MATPLOTLIB_INSTALLED, reason='matplotlib is not installed')
+def test_plot_importance(params, breast_cancer_split, train_data):
+    X_train, _, y_train, _ = breast_cancer_split
+
+    gbm0 = lgb.train(params, train_data, num_boost_round=10)
+    ax0 = lgb.plot_importance(gbm0)
+    assert isinstance(ax0, matplotlib.axes.Axes)
+    assert ax0.get_title() == 'Feature importance'
+    assert ax0.get_xlabel() == 'Feature importance'
+    assert ax0.get_ylabel() == 'Features'
+    assert len(ax0.patches) <= 30
+
+    gbm1 = lgb.LGBMClassifier(n_estimators=10, num_leaves=3, silent=True)
+    gbm1.fit(X_train, y_train)
+
+    ax1 = lgb.plot_importance(gbm1, color='r', title='t', xlabel='x', ylabel='y')
+    assert isinstance(ax1, matplotlib.axes.Axes)
+    assert ax1.get_title() == 't'
+    assert ax1.get_xlabel() == 'x'
+    assert ax1.get_ylabel() == 'y'
+    assert len(ax1.patches) <= 30
+    for patch in ax1.patches:
+        assert patch.get_facecolor() == (1., 0, 0, 1.)  # red
+
+    ax2 = lgb.plot_importance(gbm0, color=['r', 'y', 'g', 'b'],
+                              title=None, xlabel=None, ylabel=None)
+    assert isinstance(ax2, matplotlib.axes.Axes)
+    assert ax2.get_title() == ''
+    assert ax2.get_xlabel() == ''
+    assert ax2.get_ylabel() == ''
+    assert len(ax2.patches) <= 30
+    assert ax2.patches[0].get_facecolor() == (1., 0, 0, 1.)  # r
+    assert ax2.patches[1].get_facecolor() == (.75, .75, 0, 1.)  # y
+    assert ax2.patches[2].get_facecolor() == (0, .5, 0, 1.)  # g
+    assert ax2.patches[3].get_facecolor() == (0, 0, 1., 1.)  # b
+
+
+@pytest.mark.skipif(not MATPLOTLIB_INSTALLED, reason='matplotlib is not installed')
+def test_plot_split_value_histogram(params, breast_cancer_split, train_data):
+    X_train, _, y_train, _ = breast_cancer_split
+
+    gbm0 = lgb.train(params, train_data, num_boost_round=10)
+    ax0 = lgb.plot_split_value_histogram(gbm0, 27)
+    assert isinstance(ax0, matplotlib.axes.Axes)
+    assert ax0.get_title() == 'Split value histogram for feature with index 27'
+    assert ax0.get_xlabel() == 'Feature split value'
+    assert ax0.get_ylabel() == 'Count'
+    assert len(ax0.patches) <= 2
+
+    gbm1 = lgb.LGBMClassifier(n_estimators=10, num_leaves=3, silent=True)
+    gbm1.fit(X_train, y_train)
+
+    ax1 = lgb.plot_split_value_histogram(gbm1, gbm1.booster_.feature_name()[27], figsize=(10, 5),
+                                         title='Histogram for feature @index/name@ @feature@',
+                                         xlabel='x', ylabel='y', color='r')
+    assert isinstance(ax1, matplotlib.axes.Axes)
+    assert (ax1.get_title() ==
+            'Histogram for feature name {}'.format(gbm1.booster_.feature_name()[27]))
```

Comment:
```suggestion
    title = 'Histogram for feature name {}'.format(gbm1.booster_.feature_name()[27])
    assert ax1.get_title() == title
```

I think this change would address the issue from linting ():

> Linting Python code
./tests/python_package_test/test_plotting.py:91:29: W504 line break after binary operator